demo/lib/App.tsx

```diff
@@ -96,6 +103,59 @@ export default class App extends React.Component<any, AppState> {
   };
 
   render() {
+
+    let customChildren = null;
+    if (this.state.customChildren) {
+      customChildren = ({ circularProps, linearProps, ...props }) => {
+        if (this.state.viewer === "linear") {
+          return  (
+              <div ref={this.linearRef} style={{ height: "70%", width: "100%" }}>
+                <Linear {...linearProps} {...props} />
+              </div>
+          );
+        } else if (this.state.viewer === "circular") {
+          return  (
+            <div ref={this.circularRef} style={{ height: "75%", width: "100%" }}>
```

Comment:
```suggestion
            <div ref={this.circularRef} style={{ height: "100%", width: "100%" }}>
```

Same comment/question but for circular. Lmk. I'd think it'd make sense to fill the page w/ the viewer

Reply:
I was just testing to make sure 70% works.  but 100% definitely looks better!